.buildkite/pipeline.yml

```diff
@@ -139,27 +139,25 @@ steps:
     depends_on: trigger-macos
     key: macos-nix
     commands:
-      - './nix/regenerate.sh'
+      - 'nix flake info'
```

Comment:
Yes — it didn't seem to work otherwise. 😅

The purpose of the `regenerate.sh` script is to parse the list of `cabal` packages from our `cabal.project` and to write the result to `project-package-list.nix`. However, the script does this is in a self-referential manner, and I ran into trouble when the nix host is one architecture, but the compilation target is another architecture.

I think it's sufficient to run the `regenerate.sh` step on linux, and do a basic smoke test that `nix` exists on macOS instead.